src/test/java/com/github/wasiqb/coteafs/datasource/data/PropertiesData.java

```diff
@@ -0,0 +1,19 @@
+package com.github.wasiqb.coteafs.datasource.data;
+
+import com.github.wasiqb.coteafs.datasource.annotation.DataFile;
+import lombok.Data;
+
+/**
+ * @author Faisal Khatri
+ * @since Aug 31, 2020
+ *
+ */
+@Data
+@DataFile (fileName = "config.properties", folderPath = "src\\test\\resources\\properties_file")
```

Comment:
Use unix style path separator `/` instead of `\\`.

Reply:
Done